charts/sn-platform/templates/presto/presto-coordinator-configmap.yaml

```diff
@@ -63,6 +71,10 @@ data:
     #
 
     coordinator=true
+    {{- if .Values.presto.exporter }}
+    jmx.rmiregistry.port=9081
```

Comment:
are these suppose to be same? from doc it seem should be different port: https://trino.io/docs/current/admin/jmx.html

Reply:
They're different in docs, but they can be same. All remote objects exported from one JVM can share the same port with different name, including the registry itself.